src/v2/coins/abstractUtxoCoin.js

```diff
@@ -25,10 +26,6 @@ class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
-  static get validAddressTypes() {
```

Comment:
Were there no callers of this function? This is part of the public API and thus would make this pr a major version bump.

Reply:
The need for this function was removed when we started using the tcomb validation from the unspents lib. Should I add it back for the sake of backwards compatibility?